ui/app/routes/jobs/job/definition.js

```diff
@@ -31,6 +31,6 @@ export default class DefinitionRoute extends Route {
 
   setupController(controller, model) {
     super.setupController(controller, model);
-    controller.view = model.specification ? 'job-spec' : 'full-definition';
+    controller.view = model?.specification ? 'job-spec' : 'full-definition';
```

Comment:
Mentioned elsewhere but: this seems like something that might be better as a defaultValue / constructor in the definition controller, rather than the route.

Reply:
The challenge here is that the `model` is not available when the `Controller` is instantiated. Therefore using the `constructor` method of the `Controller` isn't the appropriate place. If we want `model` access, the "Ember way" would be to use `setupController`.